examples/image-with-postgis-clis/README.md

```diff
@@ -0,0 +1,10 @@
+# Using PostGIS CLIs
+The base postgis/postgis image does not have PostGIS-related CLIs enabled. To use PostGIS CLIs that are NOT enabled by default (for example `raster2pgsql`) it's necessary to extend the base image.
```

Comment:
The phrase "enabled" should probably be replaced with "installed" to be more accurate